gowin/arch.cc

```diff
@@ -589,14 +599,14 @@ Arch::Arch(ArchArgs args) : args(args)
                 const PairPOD pip = pips[p][j];
                 int destrow = row;
                 int destcol = col;
-                IdString destid(pip.dest_id);
-                IdString gdestname = wireToGlobal(destrow, destcol, db, destid);
+                IdString destid(pip.dest_id), gdestid(pip.dest_id);
+                IdString gdestname = wireToGlobal(destrow, destcol, db, gdestid);
                 if (wires.count(gdestname) == 0)
```

Comment:
Oh I see. This was changed to use dest_id directly, which is modified by wireToGlobal to be the "normalized" dest_id. Looks good then I think.